sqe-api-test/UserTests.cs

```diff
@@ -228,15 +228,17 @@ private async Task<NewUserRequestDTO> CreateRandomActivatedUserAsync(bool should
         /// <returns></returns>
         private async Task CleanupUserAccountAsync(UserDTO user)
         {
-            const string deleteNewUserSQL = "DELETE FROM user WHERE email = @Email";
-            const string deleteUserRoleSQL = "DELETE FROM users_system_roles WHERE user_id = @UserID";
-            const string deleteEmailTokenSQL = "DELETE FROM user_email_token WHERE user_id = @UserId";
+            const string deleteNewUserSql = "DELETE FROM user WHERE email = @Email";
```

Comment:
I wonder if there's a way to tell Github to ignore specific files in the code review - all of our automatically generated files can be ignored.